src/js/src/components/SearchBox.vue

```diff
@@ -135,6 +135,8 @@ export default {
         //If this is resulted by a backbutton going to the first time the user landed on the page, we want a clean slate.
         this.resetSearchState()
       }
+      document.getElementById('query').value = this.futureQuery
+      this.$_getSizeOfTextArea('query')
```

Comment:
If this can be solved with this.$refs it would be preferable. It would do the same but be more consistent with Vue